spec/amp-managing-user-state.md

```diff
@@ -0,0 +1,444 @@
+
+On AMP pages, construct an analytics-analytics ping containing the Client ID:
+
+| In ... | ... this looks like: |
+| --- | --- |
+| **amp-analytics configuration** | `https://analytics.example.com/ping?type=pageview&user_id=${clientId(uid)}` |
+| **What goes over the network** | `https://analytics.example.com/ping?type=pageview&user_id=$amp_client_id` - *In this case, `${clientId(uid)}` is replaced by an actual value that AMP either generates at that moment or will be returned based on what the user’s browser has already stored locally*|
+
+Take note of the fact that the parameter passed into the Client ID substitution, `${clientId(uid)`, is “`uid`”. This was a deliberate choice that matches the same cookie name used on the publisher origin as described in Task 1. For the most seamless integration, you should apply the same technique.
+
+Concerning the rest of the amp-analytics implementation, see the documentation for [amp-analytics configuration](https://www.ampproject.org/docs/guides/analytics_amp) for more detail on how to set up amp-analytics requests or to modify those of your analytics vendor. The ping can be further modified to transport additional data that you either directly define or by taking advantage of other [AMP substitutions](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md).
+
+> **Good to know:**
+> 
+> Why did we use of the name “`uid`” for the parameter passed to the Client ID feature? The parameter that the `clientId(...)` substitution takes is used to define scope. You can actually use the Client ID feature for many use cases and, as a result, generate many client IDs. The parameter differentiates between these use cases and so you use it to specify which use case you would like a Client ID for. For instance, you might want to send different identifiers to third parties like an advertiser and you could use the “scope” parameter to achieve this.
+
+On the publisher origin, it’s easiest to think of “scope” as what you call the cookie. By recommending a value of “`uid`” for the Client ID parameter here in Task 2, we align with the choice to use a cookie called “`uid`” in Task 1.
+
+### Task 3: Process analytics pings from pages on the publisher origin
+
+Because of the setup performed in Tasks 1 and 2, when someone accesses the AMP version (from any context) or the non-AMP version on the publisher origin the analytics ping will use the same identifier. By following the guidance in Task 2 to choose a Client ID "scope" that was the same name as the name of the cookie you used in Task 1, AMP reuses the same cookie.
+
+This is illustrated in the table below:
+
+| An analytics ping coming from... | ... looks like this: |
```

Comment:
This table would look better if you used HTML and specified width so that the 1st column isn't so squished, and hard to read.